src/f0/app_solar/source/tmp101.h

```diff
@@ -0,0 +1,234 @@
+/**
+ * @file    tmp101.h
+ * @brief   TMP101AN temperature sensor driver
+ *
+ * @addtogroup TMP101
+ * @ingroup ORESAT
+ * @{
+ */
+#ifndef _TMP101_H_
+#define _TMP101_H_
+
+#include <chprintf.h>
+
+/*===========================================================================*/
+/* Driver constants.                                                         */
+/*===========================================================================*/
+
+/**
+ * @name    Version Identification
+ * @{
+ */
+/**
+ * @brief   TMP101 Driver version string.
+ */
+#define TMP101_VERSION                      "1.0.0"
+
+/**
+ * @brief   TMP101 Driver version major number.
+ */
+#define TMP101_MAJOR                        1
+
+/**
+ * @brief   TMP101 Driver version minor number.
+ */
+#define TMP101_MINOR                        0
+
+/**
+ * @brief   TMP101 Driver version patch number.
+ */
+#define TMP101_PATCH                        0
+/** @} */
+
+
+/**
+ * @name    TMP101 Register Addresses
+ * @{
+ */
+#define TMP101_REG_TEMPERATURE_READING      0x00U
```

Comment:
The register seems to just be named "Temperature" in the datasheet, so perhaps we can just define this as TMP101_REG_TEMPERATURE

Reply:
Renamed per request.